src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,35 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { id, title, user, body, comments } = post;
+
+  return (
+    <div
+      className="PostInfo"
+      key={id}
+    >
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          <span> Posted by </span>
+
+          <UserInfo user={user} />
```

Comment:
user can be undefined, let's handle it here
```suggestion
          {user && <UserInfo user={user} />}
```